src/query_parser/v2/parser.y

```diff
@@ -504,7 +504,22 @@ attribute(A) ::= ATTRIBUTE(B) COLON param_term(C). {
       value_len = found_value_len;
     }
   }
-  A = (QueryAttribute){ .name = B.s, .namelen = B.len, .value = value, .vallen = value_len };
+  A = (QueryAttribute){ .name = rm_strndup(B.s, B.len), .namelen = B.len, .value = value, .vallen = value_len };
+}
+
+attribute(A) ::= ATTRIBUTE(B) COLON STOPWORD(C). {
```

Comment:
Let's avoid the stopword necessity by using strcmp instead of the compiler. like in attribute.

Reply:
I'm not sure how to do that, let's discuss it...

Reply:
We can use the current mechanism using `strcmp`.
If the decision is to move away from it, then your code stands.

Reply:
solved by #3130